sqlmesh/core/config/connection.py

```diff
@@ -669,6 +669,56 @@ def _connection_factory(self) -> t.Callable:
         return connect
 
 
+class MsSqlConnectionConfig(_ConnectionConfig):
```

Comment:
[Nitpick] We capitalize acronyms throughout the codebase. So it should be `MSSQLConnectionConfig`.